ui/apps/platform/src/Containers/Policies/Wizard/Form/descriptors.tsx

```diff
@@ -733,6 +733,27 @@ export const policyConfigurationDescriptor: Descriptor[] = [
         })),
         canBooleanLogic: false,
     },
+    {
+        label: 'Allow Privilege Escalation',
+        name: 'Allow Privilege Escalation',
+        shortName: 'Allow Privilege Escalation',
+        longName: 'Privilege Escalation allowed on Container',
+        category: policyCriteriaCategories.CONTAINER_CONFIGURATION,
+        type: 'radioGroup',
```

Comment:
@alwayshooin @pedrottimark 
The existing policy criterion before this has a type of `radioGroupString`, and the one after this has a type of `radioGroup`.

Should this new one have a type of `radioGroup` (as currently in the PR), or should it be `radioGroupString` ?

(I not clear on the distinction.)

Reply:
Good eyes. `type: 'radioGroup'` is correct because `value: true` and `value: false` below.

That is, `type: radioGroupBoolean` would be more descriptive. A good follow up after this is merged.

Policy criterion before is only occurrence of `type: radioGroupString` because `value` has string values:

![seccompProfileTypeLabels](https://user-images.githubusercontent.com/11862657/162221140-38423699-da10-4ba6-8dbc-caa42c614a3b.png)

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/messages/common.ts#L127-L131

Reply:
That is right, `type: radioGroupBoolean` is not there in UI currently but would be nice to add it as a separate task.